group_vars/all.yml

```diff
@@ -26,11 +26,11 @@ firewall_enabled_at_boot: true
 firewall_flush_rules_and_chains: false
 # ssh hardening
 network_ipv6_enable: true
-ssh_allow_agent_forwarding: true
-ssh_permit_tunnel: true
-ssh_allow_tcp_forwarding: 'yes'
-sftp_enabled: true
-sftp_chroot: true
+ssh_allow_agent_forwarding: "yes"
+ssh_permit_tunnel: "yes"
+ssh_allow_tcp_forwarding: "yes"
+sftp_enabled: 'yes'
```

Comment:
Is there a difference between single and double quotes? Could we agree on a coherent style?